lib/validate_params/validatable.rb

```diff
@@ -23,6 +23,29 @@ module Validatable
     class_methods do
       attr_reader :params_validations
 
+      # When controller inherits from other controller, we need to copy the validations.
+      #
+      # Usage example:
+      #
+      # class ParentController < ApplicationController
+      #   validate_params_for :index do |p|
+      #     p.param :param1, Integer, min: 1, max: 10
+      #     p.param :param2, Integer, min: 10, max: 20
+      #   end
+      # end
+      #
+      # class ChildController < ParentController
+      #   validate_params_for :index do |p|
+      #     p.param :param1, Integer, min: 0, max: 5
+      #   end
+      # end
+      #
+      # Here in ChildController, the param1 validation will be overridden, but validation for param2 will remain unchanged.
```

Comment:
@acetinick Is this good behaviour or we want to re-declare all validations if `validate_params_for` being called for particular action?